lib/flutter_client_sse.dart

```diff
@@ -23,6 +23,7 @@ class SSEClient {
       required Map<String, String> header,
       Map<String, dynamic>? body}) {
     print('---RETRY CONNECTION---');
+    if (!_streamController.isClosed)
```

Comment:
@soufianebenyaala I have removed the static _steamController causing the limitation with the access to multiple stream listeners. But this is a good suggestion. I would suggest changing the reference from _streamController to streamController.